alphametics.json

```diff
@@ -21,6 +21,16 @@
                 "description": "solve puzzle with exponents",
                 "puzzle": "PI * R ^ 2 == AREA",
                 "expected": { "P": 9, "I": 6, "R": 7, "A": 4, "E": 0 }
+            },
+            {
+                "description": "invalid solution returns null",
```

Comment:
it was good that the other case explained why it was invalid ("leading zero") - it would be good to say that here too, that A and B can't be the same digit